README.md

```diff
@@ -89,59 +92,26 @@ public void onTasksChanged() {
     }
 }
 ```
-## Additional dependencies
-
-This project uses the Loaders framework available from Android 3.0 (API Level
-11).
-
-## Features
-
-### Complexity - understandability
-
-#### Use of architectural frameworks/libraries/tools:
 
-No external frameworks.
+## Maintaining the app
 
-#### Conceptual complexity
+Adding or amending features to this version of the app requires a similar amount of effort to the todo-mvp sample. Before modifying the sample, you should be familiar with the [Loaders API](https://developer.android.com/guide/components/loaders.html), which is a relatively complex topic.
```

Comment:
I changed this in the commit to 'not trivial'. Both relatively 'simple' and 'complex' are highly subjective and depend on the experience of the reader. (relative to what?) I don't really see how either is closer to 'reality'. At least 'not trivial' implies that some effort is required.